### ISSUE TITLE
[WebToolsE2E][Aspire] After installing Aspire 8.1, the "Select a resource" option is on the wrong line in Aspire dashboard when running an Aspire Starter project

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#4767]
**REGRESSION INFO**: Work in Aspire 8.0.2

**INSTALL STEPS**

1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.302
3. Install Aspire 8.1.0-preview.1.24352.6

**REPRO STEPS**
1. Open CMD, create an Aspire Starter App using following commands:
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost

```
2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`
3. Click the "Console" option on the left, then click the "Select a resource"

**ACTUAL**
The "Select a resource" option is on the wrong line
![AspireStarter](https://github.com/dotnet/aspire/assets/88122107/c329b23d-4f00-49ac-a03f-f03df87c897f)

**Note**:
1. This issue also repro on Aspire Starter with Redis project
![AspireRedis](https://github.com/dotnet/aspire/assets/88122107/6ecdc9de-84bc-4596-8545-e1f5c81a1e1c)

**EXPECTED**
The "Select a resource" option is on the first line
![Screenshot 2024-07-03 173138](https://github.com/dotnet/aspire/assets/88122107/19afd4ba-90f1-4a51-bbbf-134c26426adb)

**Comment by balachir** [comment]
@kvenkatrajan fyi, one more dashboard issue for 8.1.

**Comment by kvenkatrajan** [comment]
@adamint  please have a look

**Comment by balachir** [comment]
@v-elenafeng my guess is that you need to wait until you have a build that includes this fix. Let's discuss in our next sync about how you can check that.

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 8.1.0-preview.1.24360.6
